compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategies.cpp

```diff
@@ -6,44 +6,23 @@
 
 #include "iree/compiler/Codegen/Common/TransformDialectStrategies.h"
 
-#include <numeric>
-#include <type_traits>
-
 #include "iree-dialects/Dialect/LinalgTransform/StructuredTransformOpsExt.h"
 #include "iree-dialects/Transforms/TransformMatchers.h"
 #include "iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.h"
+// Needed because ApplyPatternOp is used everywhere.
+// TODO: Separate out GPU-specific patterns.
 #include "iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.h"
```

Comment:
`ApplyPatternsOp` is in CommonExtensions.td/h. Are you sure this is necessary?

Reply:
you're right, addressed in a followup